systems-state-manager/nisystemsstate.yml

```diff
@@ -0,0 +1,999 @@
+  version: "1"
+  title: "SystemLink Systems State Manager"
+  contact:
+    name: National Instruments
+    url: https://www.ni.com/systemlink
+    email: support@ni.com
+basePath: "/nisystemsstate"
+x-ni-routing-key: 'Skyline.SystemsStateManager'
+x-ni-privilege-application: 'niws.nisysmgmt'
+consumes: [application/json]
+produces: [application/json]
+
+securityDefinitions:
+  basicAuth:
+    type: basic
+  cookieAuth:
+    type: apiKey
+    in: header
+    name: Cookie
+
+security:
+  - basicAuth: []
+  - cookieAuth: []
+
+responses:
+  Error:
+    description: Error
+    schema:
+      description: Error response
+      title: ErrorResponse
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+
+  BadRequest:
+    description: Bad Request indicates that the server was unable to process the request because of invalid syntax.
+    schema:
+      title: Bad Request Error Response
+      description: Bad request error response containing error information.
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+
+  Unauthorized:
+    description: Not authorized.
+    headers:
+      WWW_Authenticate:
+        description: Information for how to authenticate.
+        type: string
+
+  Conflict:
+    description: This response is sent when a request conflicts with the current state of the server.
+    schema:
+      title: Conflict Error Response
+      description: Conflict error response containing error information.
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+
+  PartialSuccess:
+    description: Partial Success in the case when only a part of the request body metadata information was processed successfully.
+    schema:
+      title: PartialSuccessResponse
+      description: Partial success response containing error information.
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+
+  NotFound:
+    description: The requested resource was not found.
+    schema:
+      title: Not Found Request Error Response
+      description: Not found error response containing error information.
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+
+definitions:
+  StateIDVersionPair:
+    type: object
+    title: StateID-Version Pair
+    properties:
+      stateID:
+        description: The ID of the state.
+        type: string
+      stateVersion:
+        description: The version of the state. The latest content of the state will be used if version is not specified.
+        type: string
+    required:
+      - stateID
+    example:
+      stateID: 5bcde471fc13379595ce9b13
+      stateVersion: 9ae65370c79ac158ad061032944afe6b94d927ca
+
+  ExportStateRequest:
+    type: object
+    title: Export State
+    properties:
+      inline:
+        description: >-
+            Whether to return the state data inline rather than as an attachment.
```

Comment:
Would it make sense to say, "Whether to return the state data inline or as an attachment."?